src/backend/joanie/tests/core/test_api_organizations_course_product_relations.py

```diff
@@ -0,0 +1,276 @@
+"""
+Test suite for Organization's CourseProductRelation API endpoint.
+"""
+
+from joanie.core import factories, models
+from joanie.tests.base import BaseAPITestCase
+
+
+class OrganizationCourseProductRelationApiTest(BaseAPITestCase):
+    """
+    Test suite for Organization's CourseProductRelation API endpoint.
+    """
+
+    def test_api_organizations_list_course_product_relations(self):
```

Comment:
Ditto about naming test convention

```suggestion
    def test_api_organizations_course_product_relations_read_list_with_accesses(self):
```